overlay.nix

```diff
@@ -130,62 +105,7 @@ in rec {
     '';
   });
 
-  qrtx = self.bashScript {
```

Comment:
Is this used by anybody? Could be cool in theory

Reply:
what is it?

Reply:
I think it prints a tx as a qr code? I guess it's useful if you're doing offline signing.

Reply:
Yes, it's for using air-gapped signing devices. I vote leave these in.

Reply:
This is cool, +1 for keep